src/main/java/com/example/quizapp/quiz/flashcard/CreateFlashcardController.java

```diff
@@ -63,6 +82,18 @@ public IQuizable<String> createQuestion(){
         return new Flashcard(frontSide.getText(), backSide.getText(), hint);
     }
 
+    public boolean isAbleToCreate() {
+        boolean ableToCreate = true;
+
+        for(var field : textFields){
+            if(!inputValidator.isValidTextField(field)){
+                ableToCreate = false;
+            }
+        }
+
+        return ableToCreate;
+    }
```

Comment:
Kan ändra till att returnera false om vi kommer in i if-satsen, annars returnera true efter loopen